extensions/fs/internal.m

```diff
@@ -1090,6 +1090,41 @@ static int fs_pathFromBookmark(lua_State *L) {
     return 1 ;
 }
 
+/// hs.fs.readFileContents(path) -> string | nil, string
+/// Function
+/// Reads the contents of a file at a given path.
+///
+/// Parameters:
+///  * path - The path to the file you want to read.
+///
+/// Returns:
+///  * A string containing the contents of the file or `nil` if an error occurs.
+///  * An error message if an error occurs.
+static int fs_readFileContents(lua_State *L) {
```

Comment:
In general, I'm not sure I like having a general "read file contents" function, given that files can contain anything, not just string values. Lua uses strings as general-purpose bit arrays as well, but this kind of implies a text value.

Reply:
Good point - I've updated the function name.